tests/bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+require dirname(__DIR__) . '/config/bootstrap.php';
```

Comment:
[original](https://github.com/symfony/recipes/blob/main/symfony/phpunit-bridge/4.3/tests/bootstrap.php) wrap this by a condition that lead to a PHPStan error.
as the file exist for sure in our case, I simplified the code to avoid that error